### DISCUSSION TITLE
Mason config TUI isn't displaying

### DISCUSSION THREAD

**Discussion by fkhawajagh** [mason-org/mason.nvim#1803] [Q&A]
I've had my nvim setup configured and working for some time.  Today I tried to launch the Mason tui but the floating window won't display.   It seems to be there and I can interact with it (blindly).

I'm using Lazy.vim and all my packages are up to date.

![image](https://github.com/user-attachments/assets/4f868469-4292-4ac2-b836-65358207bf5e)

So the ':Lazy' command works and there is a floating window.  However, when I use the command ':Mason' nothing is displayed.  Notice the floating cursor in this screenshot.

![image](https://github.com/user-attachments/assets/718db144-3068-4e13-a6e6-b6ca91aa0619)

I can scroll up and down in the floating window and even hit 'enter' to install/update package.  

![image](https://github.com/user-attachments/assets/1d2b716a-f3a1-4bee-aa87-d5df22139166)
If I'm using split buffers and then run ':Mason' I get this weird vertical sliver of a window.   I'm wondering if this is related to zindex for the floating window similar to https://github.com/williamboman/mason.nvim/issues/1787

**Comment by fkhawajagh** [comment]
Additionally, I can confirm the window is there by yanking lines from the invisible floating window and pasting them into another buffer.

**Comment by rewgs** [comment]
Exact same issue here. Did you ever get this resolved?

**Comment by 3oFiz4** [comment]
I know this is late. But there's a quick fix. I realized that Mason TUI can be only opened when you run `:only` first.  And then you can open `Mason`. For shorthand, `:only | Mason`. 

You can also keymap it.
```lua
vim.keymap.set("n", "<leader>cm", function()
  vim.cmd("only")
  vim.cmd("Mason")
end)
```

*I honestly have no idea why this works, but I believe it is probably a layout issue with Mason TUI. Because `:only` command resets the whole layout to a single grid. As if something within the layout interferes with the dimensions.*
